src/cow-react/modules/limitOrders/containers/Chart/index.tsx

```diff
@@ -0,0 +1,5 @@
+import * as styledEl from '../../pure/Chart'
```

Comment:
Respectfully I have to disagree with the rule that Containers shouldn't have any style.
I think we should tend to separate UI in pure components when it makes sense, but making this an absolute rule, in my opinion will make things harder then it needs to be just to fit that logic.

Besides as I can see we are already doing this
https://github.com/cowprotocol/cowswap/blob/bbdacab61d0c3d7be5b9f58f28f4d24b155ba148/src/cow-react/modules/swap/containers/NewSwapWidget/index.tsx#L192

However in this case, since these 2 components will grow it makes sense to do this.